_posts/2021-01-13-voicemeeter-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,385 @@
+
+Base Voicemeeter Digital Inputs:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana Digital Inputs:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Potato Digital Inputs:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Alright, seem like a lot? I agree! When I first started messing with Voicemeeter I was like... what?  But over time I was like... I can do what with this? It ended up being amazing and I still use it for certain projects. Ahem... Anyways, lets discuss the last section of a basic setup. Hardware Inputs! Again, these cover microphones, line inputs (such as a 3.5mm in) or even digital audio drivers for extra audio routing.
+
+Just like Hardware Outputs and Virtual Inputs, there are different features added in Banana and Potato than what comes with Base Voicemeeter. Base Voicemeeter gets the gain slider, a mute, mono selector ( triggers mono channeling where the same signal is sent equally through both stereo outputs. It helps some mics that for some reason only pull in through one side in some cases and it helps to balance that one sided input to both ears on output.) Banana gains a compressor and noise gate over Base Voicemeeter. Potato gains Special Effects: Reverb, Delay, and separate Sends for each. I will give an image per program to show what they look like:
+
+Base Voicemeeter Hardware Inputs:
+{
+
+
+
+  Need Picture here
+
+
+
+}
+
+Voicemeeter Banana Hardware Inputs:
+{
+
+
+
+  Need picture here
+
+
+
+}
+
+Voicemeeter Potato Hardware Inputs:
+{
+
+
+
+  Need Picture Here:
+
+
+
+}
+
+Alright, that explains the whole of voicemeter right? Go have fun with it! I just, I jest. That was mostly just trying to help you understand each part before running through a setup. When is setup? Setup is now! and it is the fun part! First off, we are going to do something different than I have seen a lot of guides do. We are going to skip around to setting up our outputs first. This way we can leave our current setup alone and you can listen to music or do other things, maybe even come say hello to use in the Library discord? [liiiink the discord here] We would be glad to ahve you and discuss things. Semi-guilt-feeling discord link aside, lets get back into setting up Voicemeeter!
```

Comment:
May be helpful to break out the start of the setup into a new section with a header, something like, "### Your First Audio Mix".

Reply:
`[StreamSounds Discord Server]({{ site.streamsounds.discord }})`

Reply:
Agreed, though the revision might already cover this as I split it up more and stuff.